### DISCUSSION TITLE
Amazing. Any plans for React Native ?

### DISCUSSION THREAD

**Discussion by dipeshparashar** [couds/react-bulma-components#392] [General]
You are doing a great job maintaining this lib. I have been following it for years now. Any plans to support React Native soon, much like these are doing [TailwindRN](https://github.com/vadimdemedes/tailwind-rn) . Bulma can be ultimate help on React Native.

**Comment by kennethnym** [comment]
I don't think Bulma supports React Native since it is a CSS framework, but I think it's certainly possible to mimic Bulma's design style and port it to React Native. Good suggestion!

**Comment by kennethnym** [comment]
@dipeshparashar I'll convert this into a discussion to see what people think of this!
